### PR TITLE
Add Announcing 2.2 blog post

### PR DESCRIPTION
**Pull request by alfonsogarciacaro**
@MangelMaxime @selketjah @Zaid-Ajaj @nojaf

This is the draft for the announcement of fable-compiler 2.2, Fable.Core 3, the new Fable.Browser packages, etc. If you have a moment, could you please have a look and tell me if you have a suggestion to make the update easier for the users?

@et1975 Would it be possible to add a paragraph for the Elmish 3 release?

Thank you!

### CODE REVIEW

**Comment by et1975** [comment]
Will do tonight.

**Comment by ncave** [comment]
@alfonsogarciacaro Since this post will (hopefully) add some visibility to the `fable-compiler-js`, do you mind publishing the latest `fable-standalone` and a new version of `fable-compiler-js` that references it?

File: blog/Announcing-2-2.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+## Fable.PowerPack
+
+Similar problems affected Fable.PowerPack. It was originally intended as an extended Fable.Core but it had become a mix of unrelated modules and many times it was difficult for maintainers to remember who had authored what.
+
+Following the reasoning above, we've decided to deprecate Fable.PowerPack and replace it with single-purpose packages. For now, [Fable.Promise](https://github.com/fable-compiler/fable-promise) and [Fable.Fetch](https://github.com/fable-compiler/fable-fetch) have been released (only v2 beta is compatible with Fable.Core 3). The API is identical to the modules in Fable.PowerPack except for these changes:
+
+- Similar to Fable.Browser, the `Fable.` prefix is removed from the actual module names. In most cases (e.g. to use `Promise.map`) you won't need to open anything.
+- `Fetch.Fetch_types` is renamed to `Fetch.Types`.
+- The fetch methods in auto-serialization (like `fetchAs`) have been removed, please combine fetching with [Thoth.Json](https://github.com/thoth-org/Thoth.Json) or [Fable.SimpleJson](https://github.com/Zaid-Ajaj/Fable.SimpleJson).
```
Comment (MangelMaxime) [inline blog/Announcing-2-2.md]: @alfonsogarciacaro Perhaps we can add a link to the old implementation of `fetchAs` so people know take a look at it and understand how to write their own version:

https://github.com/fable-compiler/fable-powerpack/blob/2459ca3772b564b68ea02d9e323e14118fc4bf5b/src/Fetch.fs#L359

**Comment by MangelMaxime** [comment]
I think we should add at the end of the section the list of the new packages replacing the old one. For example, it's not obvious for people that they need to install `Fable.Date` in order to use `Fable.PowerPack.Date` module. Same for the browser.

Other than that, the announcement sounds good and should help people understand the reason behind the changes.

**Comment by nojaf** [comment]
Looks good!

**Comment by selketjah** [comment]
agree with @MangelMaxime that a list from old to new would be handy

Might want to update `https://developer.mozilla.org/es/docs/Web/API` to the English version.

**Comment by MangelMaxime** [comment]
@alfonsogarciacaro 

After migrating my project to Fable.Core v3 and having a hard time figuring out which deps wasn't up to date I would propose to add a paragraph like that:

------------

If you see an error like this one:

```
FSC : error FS0193: The module/namespace 'Fable.Import.JS' coming from 'Fable.Core' didn't contain the namespace, module or type 'Promise`1' [/path/to/your/Project.fsproj]
```

It means that at least one of your dependencies isn't upgraded to Fable.Core v3.

One way to find the culprit is to look at your `paket.lock` file and search for `Fable.Core` line. The libraries should depend at the minimum on `Fable.Core 2.1.0-alpha-002` or `Fable.Core 3.0.0-beta-xxx`.

For example:

This library has been upgraded:
```
    Fable.Elmish.Debugger (3.0.0-beta-3)
      Fable.Core (>= 3.0.0-beta-005)
```

This library has not been upgraded:
```
    Fable.Elmish (2.0.3)
      Fable.Core (>= 2.0)
```

Please contact the maintainer of the library so he can upgrade it or send a PR to his repository explaining why it's needed.

------------

I added `Fable.Core 2.1.0-alpha-002` because of all the package from `Fable.Browser` depends on it and not `Fable.Core 3.0.0-beta-XXX`

*Note: I  manually translated the error message because it was in french and I don't know how to get it in English... thanks dotnet...*

**Comment by alfonsogarciacaro** [comment]
This is published, thank you all for your suggestions!

**Pull request merged**